malaria24/settings/base.py

```diff
@@ -33,12 +36,9 @@
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = True
 
-TEMPLATE_DEBUG = True
-
 ALLOWED_HOSTS = ['*']
 
 
-# Base URL to use when referring to full URLs within the Wagtail admin
 # backend - e.g. in notification emails. Don't include '/admin' or
 # a trailing slash
 BASE_URL = environ.get("BASE_URL") or 'http://example.com'
```

Comment:
I think you might be able to remove this setting entirely. It's not used by anything in this codebase